Nodejs/Setup/Custom_Action/CustomAction.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Deployment.WindowsInstaller;
+using System.Diagnostics;
+using System.Net;
+using System.Collections.Specialized;
+using System.Web.Script.Serialization;
+
+namespace Custom_Action
+{
+    public class CustomActions

```

Comment:
* The name of the file is CustomAction.cs
* The name of the class is CustomActions
* The name of the namespace is Custom_Action

Please make things more consistent.